UI/EmuScreen.cpp

```diff
@@ -1494,50 +1501,52 @@ void EmuScreen::render() {
 
 	Core_UpdateDebugStats(g_Config.bShowDebugStats || g_Config.bLogFrameDrops);
 
-	PSP_BeginHostFrame();
-
-	PSP_RunLoopWhileState();
-
-	// Hopefully coreState is now CORE_NEXTFRAME
-	switch (coreState) {
-	case CORE_NEXTFRAME:
-		// Reached the end of the frame, all good. Set back to running for the next frame
-		coreState = CORE_RUNNING;
-		break;
-	case CORE_STEPPING:
-	case CORE_RUNTIME_ERROR:
-	{
-		// If there's an exception, display information.
-		const MIPSExceptionInfo &info = Core_GetExceptionInfo();
-		if (info.type != MIPSExceptionType::NONE) {
-			// Clear to blue background screen
-			bool dangerousSettings = !Reporting::IsSupported();
-			uint32_t color = dangerousSettings ? 0xFF900050 : 0xFF900000;
-			thin3d->BindFramebufferAsRenderTarget(nullptr, { RPAction::CLEAR, RPAction::DONT_CARE, RPAction::DONT_CARE, color }, "EmuScreen_RuntimeError");
-			// The info is drawn later in renderUI
-		} else {
-			// If we're stepping, it's convenient not to clear the screen entirely, so we copy display to output.
-			// This won't work in non-buffered, but that's fine.
-			thin3d->BindFramebufferAsRenderTarget(nullptr, { RPAction::CLEAR, RPAction::DONT_CARE, RPAction::DONT_CARE }, "EmuScreen_Stepping");
-			// Just to make sure.
-			if (PSP_IsInited()) {
-				gpu->CopyDisplayToOutput(true);
+	bool blockedExecution = Achievements::IsBlockingExecution();
```

Comment:
Maybe this should be here and also block PSP_IsInited()?  If the game can't run, I'd rather still see a loading indicator than think PPSSPP is broken/hanging.  It should ideally block bootComplete() if it hashes the whole ISO on startup and blocks the game from running.

-[Unknown]

Reply:
Ah, sorry, this was done - just not shutdown.

-[Unknown]